public/index.html

```diff
@@ -41,19 +41,5 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-
```

Comment:
Better to split data from the public file.
Also we should keep all logic separated.